weblate/formats/exporters.py

```diff
@@ -83,7 +62,7 @@ def __init__(
         else:
             self.project = project
             self.language = language
-            self.plural = language.plural
+            self.plural = language.plural if language is not None else None
```

Comment:
Can you please explain reasons for this? The language really should be set at this point, otherwise other things might break as well.

Reply:
Actually this was done because when a subclass of BaseExporter (eg. CSVExporter) is accessed in template (glossary.html), it tries to initialize the class, even though all we need is attributes. So the class is called without any arguments, hence language is None and the above mentioned statement threw error.

Reply:
We can also do it by setting do_not_call_in_templates = True in BaseExporter. But I figured since language has default value None, there **might** be other cases too where language is not passed.

Reply:
Maybe the glossary page should probably use something like list_exporters instead, which serves exactly this purpose to avoid need to instantiate the class (the very same issue was on the translation page).

Reply:
Right. Makes sense to use list_exporters. Doing the changes.

Reply:
Thanks!